Runtime/DependencyInjectors/EditMediatorDependecyInjector.cs

```diff
@@ -36,6 +36,8 @@ public class EditMediatorDependecyInjector : MonoBehaviour
         GameObject dhvCamera;
         [SerializeField]
         ViewModeManager viewModeManager;
+        [SerializeField]
+        GameObject setupBuilding;
```

Comment:
I just realized this way is more comfortable to visualize in vscode. I regret not discovering that one year ago

```suggestion
        [SerializeField] GameObject setupBuilding;
```